server/controllers/uncategorised.js

```diff
@@ -248,7 +276,9 @@ exports.listInExAccounts = function (req, res, next) {
 
   function process(accounts) {
```

Comment:
Your logic makes sense.  However, the route is named `exports.listInExAccounts`, which is confusing because it queries more than just income/expense accounts.  In the future, please make sure your function names match what they actually do.